rita/src/rita_common/tunnel_manager/mod.rs

```diff
@@ -375,6 +377,7 @@ impl TunnelManager {
                 }
                 Err(e) => {
                     warn!("Actor mailbox failure from DNS resolver! {:?}", e);
+                    // We might need a port callback here
```

Comment:
TODO?

Reply:
maybe, I'm not sure yet, just a hunch I got while writing this but didn't want to forget.